src/routes/blogs/nimbleedge-x-onnxruntime/+page.svx

```diff
@@ -0,0 +1,137 @@
+authors:
+  [
+    'Nilotpal Pathak',
+    'Siddharth Mittal',
+    'Scott McKay',
+    'Natalie Kershaw',
+    'Emma Ning'
+    ]
+authorsLink:
+  [
+    'https://www.linkedin.com/in/nilotpal-pathak-228b97192/',
+    'https://www.linkedin.com/in/siddharth-mittal-b877a3135/',
+    'https://www.linkedin.com/in/scott-mckay-68b5ab/',
+    'https://www.linkedin.com/in/natkershaw/',
+    'https://www.linkedin.com/in/qiong-ning-21b554236/'
+
+   ]
+image: ''
+imageSquare: ''
+url: 'https://onnxruntime.ai/blogs/nimbleedge-x-onnxruntime'
+---
+
+[NimbleEdge](https://www.nimbleedge.com/) is an on-device Machine Learning (ML) platform that enables real-time personalization in mobile apps, executing data capture, processing and ML inference on end users' mobile devices vs. on cloud. Using mobile compute efficiently to deliver optimal performance with minimal device resource usage is a key priority for NimbleEdge. For this, NimbleEdge leverages various ML inference runtimes, including, prominently, **ONNX Runtime**.
+
+In this blog post, we'll explore how on-device compute can be leveraged for cost-efficient, privacy-preserving real-time ML in mobile apps, and how NimbleEdge leverages ONNX Runtime to enable this. We also share results from NimbleEdge's on-device deployment with Dream11, India's largest fantasy gaming platform with 200Mn+ users.
+
+### **Introduction**
+
+As digital consumer apps have evolved, Machine Learning (ML) has become increasingly central to digital experiences (across e-commerce, gaming, entertainment). ML-based recommendation systems are a prominent example of this phenomenon. Reducing choice paralysis and aiding conversion and engagement, these are now common in large-scale apps.
+
+Traditionally, recommendation systems utilized only historical customer behavior (e.g. purchases, views, likes), updated weekly, daily, or every few hours. However, pioneering apps such as [LinkedIn](https://www.linkedin.com/blog/engineering/member-customer-experience/near-real-time-features-for-near-real-time-personalization), [Doordash](https://doordash.engineering/2021/03/04/building-a-declarative-real-time-feature-engineering-framework/), and [Pinterest](https://medium.com/pinterest-engineering/how-pinterest-leverages-realtime-user-actions-in-recommendation-to-boost-homefeed-engagement-volume-165ae2e8cde8), have lately incorporated near-real-time and real-time features in recommendations. This has enabled highly fresh recommendations based on user interactions from just a few minutes, or even seconds ago, enabling massive uptick in key business metrics.
```

Comment:
'enabling massive uptick in key business metrics' -> 'enabling a massive uptick in key business metrics'?

Reply:
Updated.